x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_embeddable_factory.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import type { StartServicesAccessor } from 'kibana/public';
+
+import type {
+  EmbeddableFactoryDefinition,
+  IContainer,
+} from '../../../../../../src/plugins/embeddable/public';
+import { HttpService } from '../../application/services/http_service';
+import type { MlPluginStart, MlStartDependencies } from '../../plugin';
+import type { MlDependencies } from '../../application/app';
+import {
+  ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE,
+  AnomalyChartsEmbeddableInput,
+  AnomalyChartsEmbeddableServices,
+} from '..';
+import { AnomalyExplorerChartsService } from '../../application/services/anomaly_explorer_charts_service';
+
+export class AnomalyChartsEmbeddableFactory
+  implements EmbeddableFactoryDefinition<AnomalyChartsEmbeddableInput> {
+  public readonly type = ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE;
+
+  constructor(
+    private getStartServices: StartServicesAccessor<MlStartDependencies, MlPluginStart>
+  ) {}
+
+  public async isEditable() {
+    return true;
+  }
+
+  public getDisplayName() {
+    return i18n.translate('xpack.ml.components.mlAnomalyExplorerEmbeddable.displayName', {
+      defaultMessage: 'ML Anomaly charts',
```

Comment:
With the sentence case guidelines, this should be `ML anomaly charts` probably. And for the swim lane, should make it consistent, to be `ML anomaly swim lane`.